src/decimal.rs

```diff
@@ -20,7 +20,7 @@ impl Decimal {
         self.len
     }
 
-    fn to_vec(&self) -> AvroResult<Vec<u8>> {
+    pub fn to_vec(&self) -> AvroResult<Vec<u8>> {
```

Comment:
What about making this `pub(crate)`?  I am not quite sure it belongs to the public interface of the crate... Or would that be a problem?

Reply:
that should be fine.